source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,140 @@
+You would then enter the ip address of your Home Assistant OS instance as `smb://your.ha.ip.address`
+
+---
+
+### Installing and using the SSH add-on (requires enabling advanced mode for the HA user)
+
+Your Home Assistant OS install has default directories which are mapped to the add-ons so that they can be accessed by the services a particular add-on might provide. The Terminal & SSH add-on provides access to these directories over an ssh connection, and also access to nano and vi editors.  The add-on also provides access to the Home Assistant Command Line Interface (CLI) which provides custom commands for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring snapshots, and more. (See [Home Assistant via Command Line](https://www.home-assistant.io/hassio/commandline/) for further info). The Terminal & SSH add-on does *not* provide access to the underlying host file system.
+
+To use the add-on, enter a password or public key on the configuration page, then save and start the add-on.
+
+By default the Terminal & SSH addon provides a web terminal which can be added as an item to your side-panel. In order to access from an ssh client, a port needs to be entered in the network section of the add-on's configuration page.
+
+---
+
+### Installing and using VSC add-on
+
+Your Home Assistant OS install has default directories which are mapped to the add-ons so that they can be accessed by the services a particular add-on might provide. The Visual Studio Code add-on provides access to these directories through a web-based version of VSC. The add-on also provides access to the Home Assistant Command Line Interface (CLI) using VSC's terminal, which allows for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring snapshots, and more. (See [Home Assistant via Command Line](https://www.home-assistant.io/hassio/commandline/) for further info). The Visual Studio Code add-on does *not* provide access to the underlying host file system.
+
+---
+
+### Installing and using File Editor add-on
+
+---
+
+## Home Assistant OS Snapshots
+
+Snapshots are easily made in Home Assistant OS and are also available as services which can be used in automations. They can be created from the UI or the CLI (see below) and are stored in the /backup directory.
+
+A full snapshot includes the following directories:
+
+* config
+* share
+* add-ons (these are manually installed or created add-ons, and not those installed from the store)
+* ssl
+* media
+
+They also include a compressed file for each installed add-on containing the add-on's version, config settings, and other data if any.
+
+### Making a Snapshot from the UI
+
+1. Go to Supervisor > Snapshots in the UI
+2. Provide a name for the snapshot.
+3. Choose full or partial.
+4. Choose to password protect or not.
+
+> Warning - Password protected snapshots cannot easily be browsed outside of Home Assistant OS
```

Comment:
This doesn't render as a proper warning. Also probably shouldn't be a warning this big, just add it to step 4.